src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1717,6 +1717,44 @@ public override IOperation VisitCaughtException(ICaughtExceptionOperation operat
             throw ExceptionUtilities.Unreachable;
         }
 
+        public override IOperation VisitInvocation(IInvocationOperation operation, int? captureIdForResult)
+        {
+            if (operation.Instance != null)
+            {
+                _evalStack.Push(Visit(operation.Instance));
+            }
+
+            ImmutableArray<IArgumentOperation> arguments = operation.Arguments;
+            foreach (IArgumentOperation argument in arguments)
+            {
+                _evalStack.Push(Visit(argument.Value));
+            }
+
+            ImmutableArray<IArgumentOperation> visitedArguments;
+            if (!arguments.IsEmpty)
+            {
+                var builder = ArrayBuilder<IArgumentOperation>.GetInstance();
```

Comment:
>GetInstance() [](start = 63, length = 13)

Do we know how many items we plan to add?

Reply:
Yes, I'll address this in a follow up PR. I'm creating a helper for pushing and popping from the stack we will be doing it regularly.